dockerfiles/Dockerfile_Ubuntu16.04.run.x86_64

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:16.04
+
+# Update image
+SHELL ["/bin/bash", "-c"]
+
+RUN apt-get update && apt-get install -y --no-install-recommends libcurl4-openssl-dev python3-pip python3 libboost-python-dev python3-dev
+
+WORKDIR /usr/sdk
+
+RUN pip3 install --upgrade pip
+RUN pip install azure-iothub-device-client
+
+# Copy code (this assumes the ./src folder contains the code using the SDK and that the entrypoint is app.py)
+COPY ./src ./src
+
+CMD ["python3", "-u", "./src/app.py"]
```

Comment:
It looks like this is for running a .py on Ubuntu when the whl has already been installed?  Does this sample add anything where we'd explicitly need it? -- if it does could we comment on it a bit more here and/or in the readme.md?

Reply:
correct. I still need to address this in the readme.